server/app/app.go

```diff
@@ -250,11 +249,25 @@ func (a *App) Run() {
 	a.grpcServer = grpc.NewServer(opts...)
 
 	pb.RegisterGRPCForwarderServer(a.grpcServer, a.Server)
-	if err := a.grpcServer.Serve(listener); err != nil {
-		log.Panic(err.Error())
-	}
-}
+	var stopChan = make(chan os.Signal, 2)
```

Comment:
nit: do we really need this to be a buffered channel?
```suggestion
	var stopChan = make(chan os.Signal)
```

Reply:
Since we can receive messages for 2 signals, the IDE recommends it to be bufferd